REFERENCE.md

```diff
@@ -23,6 +23,7 @@ _Public Classes_
 Currently only the install from url is implemented, when Prometheus will deliver packages for some Linux distros I will
 implement the package install method as well
 The package method needs specific yum or apt repo settings which are not made yet by the module
+* [`prometheus::memcached_exporter`](#prometheusmemcached_exporter): This module manages prometheus node memcached_exporter
```

Comment:
Why is there memcached stuff

Reply:
the reference.md is generated for all classes in this module. during changes on the memchaced_exporter class, the file wasn't regenerated.

Reply:
okay. can't be changed? delete the line and extra PR for this line?